src/operator/controllers/secrets/types/secret_types.go

```diff
@@ -0,0 +1,90 @@
+package secretstypes
+
+import (
+	"context"
+	"github.com/samber/lo"
+	"strings"
+)
+
+type SecretType string
+type CertType string
+
+const (
+	TlsSecretType = SecretType("TLS")
+	JksCertType   = CertType("jks")
+	PemCertType   = CertType("pem")
+)
+
+func StrToCertType(strCertType string) CertType {
+	switch CertType(strings.ToLower(strCertType)) {
+	case JksCertType:
+		return JksCertType
+	case PemCertType:
+		return PemCertType
+	default:
+		return PemCertType
```

Comment:
If it's possible to get an unsupported value, it should return an error (if it comes from the user) or panic (if it comes from code and should never happen). If you get an unexpected value and behave as if you got PEM, that's unexpected and surprising behavior